src/write_gcm.c

```diff
@@ -4183,6 +4193,10 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
           metadata);
     wg_payload_destroy(to_remove);
   }
+  if (drop_count != 0) {
```

Comment:
Note that the metrics being dropped are from the back of the queue, not the front of the queue. So it's very likely that the new batch of metrics *will* remain on the queue, unless we actually publish 100k metrics in one minute.
Also note that this method is writing exactly 1 metric at a time, so this warning will be printed for each metric that we drop -- is this what we want?